geostackr.py

```diff
@@ -330,6 +346,53 @@ def save_plots_and_get_urls(top_list: List[Tuple[str, UserScores]], series_index
     return formatted_urls
 
 
+def add_new_series_to_yaml(name, author, regex):
+    series_dict = {
+                     "title" : name,
+                     "author" : author,
+                     "regex" : regex
+                  }
+
+    with open('series.yaml', 'r') as yamlfile:
+        current = yaml.safe_load(yamlfile)
+        current['series'].append(series_dict)
+
+    if current:
+        with open('series.yaml', 'w') as yamlfile:
+            yaml.safe_dump(current, yamlfile)
+            print("series " + name + " by " + author + " has been added")
+
+def check_for_new_series():
+    print('checking for new series to be tracked...')
+    reddit = get_reddit_instance()
+    subreddit = reddit.subreddit('soulofdestiny')
```

Comment:
Good idea! Should probably add this to the `config.yaml` so different subs can be used for testing